doc/doc.md

```diff
@@ -35,44 +35,39 @@ Options:
 - `namespace`: [`<Map>`] hash of interfaces
 - `text`: [`<string>`] data to parse
 
-_Returns:_ [`<Map>`] hash of hash of records, `{ title, description, parameters,
-    comments }`
+_Returns:_ [`<Map>`] hash of hash of records, `{ title, description, parameters, comments }`
 
 Introspect interface
 
-
 #### parseSignature(fn, text, start)
 
-- `fn`: [`<Function>`]` | `[`<RegExp>`] to be searched
+- `fn`: [`<Function>`]`|`[`<RegExp>`] to be searched
 - `text`: [`<string>`] to be searched in
 - `start`: [`<number>`] position to start searching from
 
 _Returns:_ [`<Object>`] function signature
+
 - `title`: [`<string>`] short function description
 - `description`: [`<string>`] extended function description
-- `parameters`: [`<Object[]>`][`<Object>`] function parameters, `{ name, types,
-      nonStandardTypes, comment, offset }`
-- `comments`: [`<Object[]>`][`<Object>`] comments about returned value, thrown
-      errors, deprecation and usage, `{ name, types, nonStandardTypes, comment
-      }`
+- `parameters`: [`<Object[]>`][`<object>`] function parameters, `{ name, types, nonStandardTypes, comment, offset }`
+- `comments`: [`<Object[]>`][`<object>`] comments about returned value, thrown
+  errors, deprecation and usage, `{ name, types, nonStandardTypes, comment }`
```

Comment:
All lines except for the first one are supposed to have padding. Also, why prettier didn't merge this line with the previous one?